docs/en/observability/monitor-uptime.asciidoc

```diff
@@ -3,23 +3,23 @@
 
 [NOTE]
 =====
-If you haven't already, you need to install and configure {heartbeat} to populate
+If you haven't already, please install and configure {heartbeat} to populate
```

Comment:
I would keep the original. It stresses how important it is to install and configure Heartbeat; otherwise, you will have no data to analyze.